### PR TITLE
Site: volunteer page

### PR DESCRIPTION
**Pull request by aapeliv**

### CODE REVIEW

File: site/markdown/contribute.md [hunk 1]
```diff
@@ -1,34 +1,32 @@
 ---
 title: "Join the team: become a Contributor"
-crumb: Volunteer
+crumb: Contribute
```
Comment (ItsiW) [inline site/markdown/contribute.md:3]: Buttons everywhere are going to say "Volunteer"

I think our wording should be:
A Contributor volunteers on the team

So noun is "contributor" and verb is "to volunteer" (avoid verb "to contribute")

File: site/markdown/contribute.md [hunk 2]
```diff
@@ -1,34 +1,32 @@
...
-Contributors on our team develop their skills, mentor others, and work and make friends with a global network of highly skilled and interesting individuals—all while working on a high-impact and rapidly moving project that they love.
+Contributors on our team develop their skills, mentor others, and work and make friends with a global network of highly skilled and interesting individuals — all while working on a high-impact and rapidly moving project that they love.
 
 ## Open positions
 
+- Mobile developers (React Native, Flutter, or native)
 - Frontend engineers (React and TypeScript)
-- Backend engineers (Python)
-- Mobile (iOS and Android) developers
+- Backend engineers (Python, SQLAlchemy)
```
Comment (ItsiW) [inline site/markdown/contribute.md:17]: Are we sure we want to split the terminology? Here Backend + Frontend are engineers but Mobile are developers

**Review by ItsiW — CHANGES_REQUESTED** [review]

File: site/markdown/contribute.md (hunk 2 above)
Comment (aapeliv) [reply site/markdown/contribute.md:17]: I didn't do that on purpose

File: site/markdown/contribute.md (hunk 1 above)
Comment (aapeliv) [reply site/markdown/contribute.md:3]: Sure, also note I renamed the page

**Pull request merged**